cyctl/internal/describe/template_auth_rules.go

```diff
@@ -0,0 +1,76 @@
+package describe
+
+import (
+	"fmt"
+
+	"github.com/cyclops-ui/cyclops/cyclops-ctrl/api/v1alpha1/client"
+	"github.com/cyclops-ui/cycops-cyctl/internal/kubeconfig"
+	"github.com/cyclops-ui/cycops-cyctl/utility"
+	"github.com/spf13/cobra"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var (
+	describeTemplateAuthRuleExample = ` 
+	# Describe a single templateauthrule
+	cyctl describe templateauthrule templateauthrule1
+ 
+	# Describe multiple templateauthrule
+	cyctl describe templateauthrule templateauthrule1 templateauthrule2 templateauthrule3`
+)
+
+// describeTemplateAuthRules describe a specified templateauthrules from the Cyclops API.
+func describeTemplateAuthRules(clientset *client.CyclopsV1Alpha1Client, templateAuthNames []string) {
+	if len(templateAuthNames) == 0 {
+		templateauthrules, err := clientset.TemplateAuthRules("cyclops").List(v1.ListOptions{})
+		if err != nil {
+			fmt.Printf("Error fetching templateauthrules: %v\n", err)
+			return
+		}
+		for _, templateAuth := range templateauthrules {
+			templateAuthNames = append(templateAuthNames, templateAuth.Name)
+		}
+	}
+
+	for _, templateAuthName := range templateAuthNames {
+		templateAuth, err := clientset.TemplateAuthRules("cyclops").Get(templateAuthName)
+		if err != nil {
+			fmt.Printf("Error from server (NotFound): %v\n", err)
+			return
+		}
+		// Describe the templateAuthRule
+		descirbe := utility.Describe(func(d *utility.Describer) {
```

Comment:
Just saw a typo here
```suggestion
		describe := utility.Describe(func(d *utility.Describer) {
```